monolog/composer.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "friendica-addons/monolog",
+  "description": "Monolog can send your logs to files, sockets, inboxes, database, etc..",
+  "type": "friendica-addon",
+  "authors": [
+	{
+	  "name": "Philipp Holzer",
+	  "email": "admin@philipp.info",
+	  "homepage": "https://blog.philipp.info",
+	  "role": "Developer"
+	}
+  ],
+  "require": {
+	"php": ">=7.0",
+    "monolog/monolog": "^3.2"
+  },
```

Comment:
Mismatched indentation through the file and you probably should declare the PSR-4 namespace `\Fusion\Addon\monolog\src` here.